server/channels/app/export.go

```diff
@@ -1155,15 +1171,19 @@ func (a *App) exportFile(outPath, filePath string, zipWr *zip.Writer) *model.App
 				nil, "err="+err.Error(), http.StatusInternalServerError)
 		}
 
-		wr, err = os.Create(filePath)
+		wr, err := os.Create(filePath)
```

Comment:
No need for a new variable
```suggestion
		wr, err = os.Create(filePath)
```